content/docs/how-we-work/git/git-pr.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Pull Request (PR)
+weight: 2
+description: A Git PR, or pull request, is a way to propose changes to the codebase. Post review your changes are merged. This process helps to ensure that changes are reviewed by someone else, which reduces the chances of any errors.
+---
+
+# Pull Request (PR)
+
+A Git PR, or pull request, is a way to propose changes to the codebase. Post review your changes are merged. This process helps to ensure that changes are reviewed by someone else, which reduces the chances of any errors.
```

Comment:
Let’s make it clear that PR is a GitHub or GitLab feature (called MR over there). Git doesn’t need anything called pull requests.